spec/order_spec.rb

```diff
@@ -4,7 +4,6 @@
 RSpec.describe Order do
   describe '#place order' do 
     it 'places order' do
-      menu = Menu.new
       expect(subject.place_order(:chips, :lamb_shish, :soft_drink)).to eq [3,10,2]
```

Comment:
really like that your place order returns the cost! Could customers place their order as a string?